src/main/java/com/cloudbees/jenkins/support/impl/ProcFilesRetriever.java

```diff
@@ -65,11 +63,11 @@ public void addContents(@NonNull Container container) {
 
     protected void addUnixContents(@NonNull Container container, final @NonNull Node node) {
         Computer c = node.toComputer();
-        if (c == null) {
+        if (c == null || c.isOffline()) {
             return;
         }
         // fast path bailout for Windows
-        if (c instanceof SlaveComputer && !Boolean.TRUE.equals(((SlaveComputer) c).isUnix())) {
+        if (!Boolean.TRUE.equals(c.isUnix())) {
```

Comment:
This and above seem unrelated?

Reply:
You mean any refactoring due to new baseline should be in the separate PR?

Reply:
Yes. For future PRs.

Reply:
It should be refactored when the new baseline is updated.